### PR TITLE
TA#21056 Add module web_handle_condition

### PR DESCRIPTION
**Pull request by ddufresne**

### CODE REVIEW

File: web_handle_condition/static/src/js/handle.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        if(this._isHandleInvisible()){
+            this._hideHandle();
+        }
+        else {
+            this._showHandle();
+        }
+    },
+
+    _isHandleInvisible() {
+        return Boolean(this.record.data[handleInvisibleField]);
```
Comment (ddufresne) [inline web_handle_condition/static/src/js/handle.js:32]: Codacy Issue found: [Function Call Object Injection Sink](https://app.codacy.com/manual/numigi/odoo-web-addons/pullRequest?prid=5169148)

File: web_handle_condition/static/src/js/handle.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    _render() {
+        this._super.apply(this, arguments);
+
+        if(this._isHandleInvisible()){
+            this._hideHandle();
+        }
+    },
+
+    _reset() {
+        this._super.apply(this, arguments)
```
Comment (ddufresne) [inline web_handle_condition/static/src/js/handle.js]: Codacy Issue found: [Missing semicolon.](https://app.codacy.com/manual/numigi/odoo-web-addons/pullRequest?prid=5169148)

File: web_handle_condition/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    'name': 'Web Handle Condition',
+    'version': '1.0.0',
+    'author': 'Savoir-faire Linux',
```
Comment (foutoucour) [inline web_handle_condition/__manifest__.py]: Le 🐧 est impliqué dans l'histoire ?

Comment (ddufresne) [reply web_handle_condition/__manifest__.py]: nope, erreur de copier-coller

Comment (ddufresne) [reply web_handle_condition/__manifest__.py]: @foutoucour fixed

Comment (foutoucour) [reply web_handle_condition/__manifest__.py]: je me disais :D

**Review by foutoucour — APPROVED** [review]

**Pull request merged**